docs/pages/includes/node-logins.mdx

```diff
@@ -0,0 +1,30 @@
+When Teleport's Auth Service receives a request to list Teleport Nodes (e.g., to
+display Nodes in the Web UI or via `tsh ls`), it only returns the Nodes that the
+current user is authorized to access.
+
+For each Node in the user's Teleport cluster, the Auth Service applies the
+following checks in order and, if one check fails, hides the Node from the user:
+
+{/*
+
+ TODO: We might want to mention that the Auth Service checks the resource's
+ namespace as well, but we currently do not document resource namespaces.
+ This would not be the appropriate place to include our only namespace
+ documentation. See gravitational/teleport issue #12580.
+ 
+*/}
+
+- None of the user's roles contains a `deny` rule that matches the Node's labels.
+- None of the user's roles contains a `deny` rule that matches the user's
+  `traits.logins`.
+- At least one of the user's roles contains an `allow` rule that matches the
+  Node's labels.
+- At least one of the user's roles contains an `allow` rule that matches the
```

Comment:
I think mentioning traits makes it a bit more confusing. I would just say that at least one of user's roles has allowed logins for the node. Traits aren't really relevant in this context I think.